service-deployment/src/webportal/dockerfile

```diff
@@ -24,7 +24,7 @@ ENV NODE_ENV=production \
 
 COPY copied_file/ /usr/src/
 COPY copied_file/webportal/ /usr/src/app/
-RUN npm install
+RUN npm run yarn install
```

Comment:
Is it really necessary to use yarn? I tried `npm install` and it worked fine.

Reply:
of course, we use yarn as package manager.
you can find the doc [here](https://yarnpkg.com/en/docs/migrating-from-npm)

Reply:
[https://www.sitepoint.com/yarn-vs-npm/](https://www.sitepoint.com/yarn-vs-npm/) here are more info about why yarn is better than npm

---
In reply to: [158422658](https://github.com/Microsoft/pai/pull/124#discussion_r158422658) [](ancestors = 158422658)